java-frontend/src/test/java/org/sonar/java/SonarComponentsTest.java

```diff
@@ -218,16 +216,14 @@ public void no_issue_when_check_not_found() throws Exception {
     JavaCheck expectedCheck = new CustomCheck();
     CheckRegistrar expectedRegistrar = getRegistrar(expectedCheck);
 
-    Issuable issuable = mock(Issuable.class);
-
     when(this.checks.ruleKey(any(JavaCheck.class))).thenReturn(null);
     SonarComponents sonarComponents = new SonarComponents(fileLinesContextFactory, null, null, null, checkFactory, new CheckRegistrar[] {
       expectedRegistrar
     });
     sonarComponents.setSensorContext(context);
 
     sonarComponents.addIssue(new File(""), expectedCheck, 0, "message", null);
-    verify(issuable, never()).addIssue(any(Issue.class));
```

Comment:
Am i right that this change should have happened a long time ago?

Reply:
Yep, test was not testing anything